numpy/_core/tests/examples/cython/meson.build

```
@@ -34,3 +34,17 @@ py.extension_module(
     ],
     include_directories: [npy_include_path],
 )
+
+py.extension_module(
+    'limited_api',
+    'limited_api.pyx',
+    install: false,
+    c_args: [
+      '-DNPY_NO_DEPRECATED_API=0',  # Cython still uses old NumPy C API
+      # Require 1.25+ to test datetime additions
+      '-DNPY_TARGET_VERSION=NPY_2_0_API_VERSION',
+      '-DCPYTHON_LIMITED_API=1',
```

```suggestion
      '-DCYTHON_LIMITED_API=1',
```

(Assuming you don't take my suggestion to exclude Cython here for now)

---

Thanks. With that fix and cython 3.0.6 the test passes locally